packages/google_atlas/pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: google_atlas
 description: Google Map Provider for Atlas, an extensible map abstraction for Flutter with support for multiple map providers
-version: 0.3.5
+version: 0.4.0
```

Comment:
the `CHANGELOG.md` has the version 0.3.6 and here is 0.4.0

Reply:
> the `CHANGELOG.md` has the version 0.3.6 and here is 0.4.0

My bad, nice catch, It was fixed